profiles/acrossx.jsonld

```diff
@@ -13,9 +13,8 @@
   },
   "versions": [
     {
-      "id": "https://w3id.org/xapi/acrossx/v1",
-      "generatedAtTime": "2017-07-14T11:35:00Z",
-      "wasRevisionOf": ["https://w3id.org/xapi/acrossx"]
+      "id": "https://w3id.org/xapi/acrossx",
```

Comment:
You can't use the same identifier for the profile as a whole and a profile version.